smartmin/views.py

```diff
@@ -141,10 +141,6 @@ def lookup_obj_attribute(self, obj, field):
         # next up is the object itself
         obj_field = getattr(obj, curr_field, None)
 
-        # if it is callable, do so
-        if obj_field and getattr(obj_field, '__call__', None):
```

Comment:
Fair nuff - so what shall we do about m2m fields? They currently cause `SmartReadView` and `SmartListView` to blow up. We can either 1) fetch the related objects and generate a value from their csv'd str values, or 2) blow up with a more helpful message than https://sentry.io/nyaruka/casepro/issues/288024834/ so dev knows to add a method to the view or exclude that field.